proxy/proxy-cms-admin-dev.conf

```diff
@@ -20,47 +20,79 @@
 #     - run: yarn dev
 #   4. visit http://localhost:8081/admin/
 #
-server {
-    listen 8081 default_server;
-    server_name _;
-    access_log /dev/stderr combined;
-    error_log /dev/stderr warn;
+pid /tmp/nginx/pid;
 
-    # let hugo handle requests by default
-    # (a possible webhook for updates also goes here)
-    location / {
-        proxy_pass https://handbook.giantswarm.io;
-        proxy_pass_request_headers      on;
-        proxy_set_header   X-Real-IP        $remote_addr;
-        proxy_set_header   X-Forwarded-For  $proxy_add_x_forwarded_for;
-        proxy_set_header Early-Data $ssl_early_data;
-    }
+error_log /dev/stdout warn;
 
-    # proxy /admin, /auth and /callback to the docker-decap-cms-standalone container
-    location /admin/ {
-        proxy_pass http://host.containers.internal:8080/;   # target: static-cms dev server (live reload), modify for docker
-        proxy_pass_request_headers      on;
-        proxy_set_header   Host        localhost:8081;
-        proxy_set_header   X-Real-IP        $remote_addr;
-        proxy_set_header   X-Forwarded-For  $proxy_add_x_forwarded_for;
-        proxy_set_header Early-Data $ssl_early_data;
-    }
+worker_processes 1;
 
-    location /auth {
-        proxy_pass http://host.containers.internal:8082;
-        proxy_pass_request_headers      on;
-        #proxy_set_header   Host        localhost:8081;
-        proxy_set_header   X-Real-IP        $remote_addr;
-        proxy_set_header   X-Forwarded-For  $proxy_add_x_forwarded_for;
-        proxy_set_header Early-Data $ssl_early_data;
-    }
+events {
+  worker_connections 1024;
+}
+
+http {
+    include /etc/nginx/mime.types;
+    default_type application/octet-stream;
+    open_file_cache max=1000 inactive=60s;
+    open_file_cache_valid 30s;
+    open_file_cache_min_uses 2;
+    open_file_cache_errors on;
+    sendfile on;
+    server_tokens off;
+    log_format custom '"$request" '
```

Comment:
Quite strange non-standard log format

[opinion, not a request for change]

Reply:
What we use for Intranet. What would be our default?

Reply:
Created a follow up issue: https://github.com/giantswarm/giantswarm/issues/32205